ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksPage.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { Button, PageSection } from '@patternfly/react-core';
+
+import { complianceEnhancedCoveragePath } from 'routePaths';
+
+import CoveragesToggleGroup from './CoveragesToggleGroup';
+import CoveragesPageHeader from './CoveragesPageHeader';
+
+function ProfileChecksPage() {
+    const history = useHistory();
+    const { profileName } = useParams();
+
+    return (
+        <>
+            <CoveragesPageHeader />
+            <PageSection>
+                <CoveragesToggleGroup tableView="checks" />
+            </PageSection>
+            <PageSection variant="light">
+                <div>ProfileChecksPage</div>
+                <Button
+                    onClick={() => {
+                        history.push(
+                            `${complianceEnhancedCoveragePath}/profiles/${profileName}/clusters`
+                        );
+                    }}
+                    variant="primary"
+                >
+                    Go to all clusters (ProfileClustersPage)
+                </Button>
+                <Button
+                    onClick={() => {
+                        history.push(
+                            `${complianceEnhancedCoveragePath}/profiles/${profileName}/checks/test`
+                        );
+                    }}
```

Comment:
Although `ToggleGroupItem` supports only an imperative `history.push(…)` call, let’s prefer React Router `Link` element whenever possible when we replace these demo stubs with tables, and so on.

Reply:
Agreed, unfortunately this same component will be used for switching between checks and clusters so we're going to have to do the same there.